tests/unit/test_general_options.py

```diff
@@ -438,20 +439,20 @@ class       Command group for CIM classes.
      {'general': ['--mock-server', 'fred'],
       'cmdgrp': 'connection',
       'args': ['show']},
-     {'stderr': ['--mock-server: File:',
+     {'stderr': ['Error: Mock file',
```

Comment:
Comment on "Error:"

Reply:
Tried to keep click part of text out of as many messages as possible to avoid our tests getting caught in their possible changes.

Reply:
Leave as is